py/picca/delta_extraction/astronomical_objects/forest.py

```diff
@@ -523,6 +523,10 @@ def rebin(self):
             w1 = w1 & (self.lambda_ / (1. + self.z) < Forest.lambda_max_rest_frame)
             w1 = w1 & (self.ivar > 0.)
             if w1.sum() == 0:
+                self.lambda_ = np.array([])
```

Comment:
@iprafols The issue I was having in `compute_delta_stack` was due to a short forest not being rejected for `wave solution = lin`, but it was for `wave solution = log`, for which there was some negative flux bins... and the differences comes from those lines being present or not. Were those lines really missing?

Reply:
@alxogm, yes they were really missing. Good detective work!

Reply:
I'll add a commit to the delta extraction branch with this